app/src/data/servey.ts

```diff
@@ -0,0 +1,99 @@
+export const mySurveys = [
+  {
+    id: 1,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 82,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+    gift: 'etc',
+  },
+  {
+    id: 2,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 50,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+    gift: 'coffee',
+  },
+  {
+    id: 3,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 50,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+  },
+];
+
+export const interestSurveys = [
+  {
+    id: 1,
+    title: '제로(0kcal)음료에 대한 인식',
+    tag: '#20~30대  #일상',
+  },
+  {
+    id: 2,
+    title: '‘그린워싱’ 제품 사용 경험에 대한 설문조사',
+    tag: '#20~30대  #사회',
+    gift: 'etc',
+  },
+  {
+    id: 3,
+    title: '헬스케어 플랫폼 설문조사',
+    tag: '#20~30대  #IT',
+  },
```

Comment:
없는 값은 null 로 명시해주세요!
```suggestion
  {
    id: 3,
    title: '헬스케어 플랫폼 설문조사',
    tag: '#20~30대  #IT',
    gift: null
  },
```